test/JDBC/input/views/sys-type_info_internal-vu-prepare.sql

```diff
@@ -0,0 +1 @@
+GO
```

Comment:
We can create a dummy table here just to ensure the db is not empty

Reply:
So this test is testing the sys.type_info_internal view which is 'static' and should return the same values regardless of the database state.
Also, by default, this test is connecting to the master database which is not empty.

However, if you feel that creating a dummy table here would help, I can definitely add it.
Should I?

Reply:
Added prepare/cleanup steps after removing permission from the sys.table_types_internal view.